### PR TITLE
Test with Python 3.6

### PR DESCRIPTION
**Pull request by jonparrott**

### CODE REVIEW

**Comment by jonparrott** [comment]
It can run `py36`, we just need a grpc wheel.

Waiting on https://github.com/grpc/grpc/issues/9445

File: scripts/travis.sh [hunk 1]
```diff
@@ -25,6 +25,9 @@ if [[ -n $SYSTEM_TEST ]]; then
     if [[ $TRAVIS_SECURE_ENV_VARS == "true" ]]; then
         echo 'Extracting secrets.'
         scripts/decrypt-secrets.sh "$SECRETS_PASSWORD"
+        # Prevent build failures from leaking our password.
+        # looking at you, Tox.
```
Comment (dhermes) [inline scripts/travis.sh]: DERP

**Comment by dhermes** [comment]
Still LGTM, pending the wheel

**Comment by jonparrott** [comment]
> Still LGTM, pending the wheel

Thanks. :)

**Comment by jonparrott** [comment]
PyLint doesn't yet work on 3.6, so we're going to hold that env back to 3.5

https://github.com/PyCQA/pylint/issues/1072

File: scripts/travis.sh (hunk 1 above)
Comment (The-Compiler) [inline scripts/travis.sh]: Ouch... I opened https://github.com/tox-dev/tox/issues/466 about that.

**Comment by jonparrott** [comment]
Thanks.

**Pull request merged**